lib/query/log-warnings.js

```diff
@@ -1,5 +1,5 @@
 const _ = require('lodash')
-const Utils = require('../utils')
+const { getDataTypeFromValue } = require('../helpers')
```

Comment:
This change allows us to get rid of now obsolete `utils` as well as the `moment` dependency.